[Term Entry] Dart map.containsKey() (#4445)
Codecademy/docs: content/dart/concepts/map/terms/containsKey/containsKey.md

---

> Title: '.containsKey'

Reviewer: ```suggestion
Title: '.containsKey()'
```

Description: 'Checks if a particular key is present in a Map object.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Dart'
  - 'Properties'
  - 'Map'
CatalogContent:
  - 'learn-dart'
  - 'paths/computer-science'
---

> In Dart, the **`.containsKey()`** method is an efficient and quick way to check if a `Map` object contains a specific key.

Reviewer: ```suggestion
In Dart, the **`.containsKey()`** method is an efficient and quick way to check if a particular key is present in a `Map` object.
```

## Syntax

```pseudo
Map-variable.containsKey(key)
```

> - `Map-variable`: Represents a variable that is a `Map` object.
> - `key`: Represents the key to be checked within the `Map`.

Reviewer: ```suggestion
- `mapVariable`: The name of the `Map` object to be checked.
- `key`: The key to be checked within the `Map` object.
```

> ## Example 1
> 
> In the following example, the `.containsKey()` method is used to check if the keys are present in the `Map` object.

Reviewer: ```suggestion
## Examples

In the following example, the `.containsKey()` method is used to check if the specified keys are present in the `Map` object:
```
